test/bigtest/tests/package-content-test.js

```diff
@@ -0,0 +1,126 @@
+import { beforeEach, describe, it } from '@bigtest/mocha';
+import { faker } from '@bigtest/mirage';
+
+import { expect } from 'chai';
+
+import setupApplication from '../helpers/setup-application';
+import PackageContentInteractor from '../interactors/package-content';
+import BasketInteractor from '../interactors/basket';
+import AgreementFormInteractor from '../interactors/agreement-form';
+import AgreementViewInteractor from '../interactors/agreement-view';
+
+const PKG = {
+  name: 'access_start_access_end_tests Package'
+};
+
+const currentResource = {
+  'pti': { titleInstance: { id: '1234', name: 'Accounts of chemical research' } },
+  'accessStart': () => faker.date.recent().toISOString(),
+};
+
+const futureResource = {
+  pti: { titleInstance: { id: '234', name: 'Analytical chemistry' } },
+  accessStart: () => faker.date.future().toISOString()
+};
+
+const droppedResource = {
+  pti: { titleInstance: { id: '734', name: 'Bioconjugate chemistry' } },
+  accessEnd: () => faker.date.recent().toISOString()
+};
+
+describe('Package Content Filters', () => {
+  setupApplication();
+  const packagecontent = new PackageContentInteractor();
+  const basket = new BasketInteractor();
+  const agreementform = new AgreementFormInteractor();
+  const agreementview = new AgreementViewInteractor();
+
+  let eresource;
+
+  beforeEach(async function () {
+    eresource = this.server.create('eresource', { name: PKG.name, class: 'org.olf.kb.Pkg', pcis: [droppedResource, currentResource, futureResource] });
+  });
+
+  describe('packagecontent tests', () => {
+    beforeEach(async function () {
+      this.visit(`/erm/eresources/${eresource.id}`);
+    });
+
+    describe('visiting the eresource pane', () => {
+      it('should render expected package', () => {
+        expect(packagecontent.headline).to.equal(PKG.name);
+      });
+
+      it('should render the expected current eresource', () => {
+        expect(packagecontent.eresourceName).to.equal(currentResource.pti.titleInstance.name);
+      });
+
+      describe('clicking the future eresource tab', () => {
+        beforeEach(async function () {
+          await packagecontent.clickFuture();
+        });
+
+        it('should render the expected future resource', () => {
+          expect(packagecontent.eresourceName).to.equal(futureResource.pti.titleInstance.name);
```

Comment:
There's a big mix of all lowercase and camelCase in these tests. Is there a style reason why you tend to lowercase the interactor variable names?

Reply:
No specific reason. Will just change the interactor variables to camelCase